doc/ref_model/chapters/chapter03.md

```diff
@@ -539,6 +542,150 @@ SyncE was standardized by the ITU-T, in cooperation with IEEE, as three recommen
+                                        labels:
+                                                app: my-app
+                        spec:
+                                containers:
+                                        - name: my-app
+                                          image: my-app-1.2.3
+                                          ports:        
+                                          - containerPort: 123 
+```
+
+This is all that is needed to deploy 4 pods/containers that are fronted by a service that performe sload balancing. The *Deplyment* will ensure that there are always four pods of type *my-app* available. the *deplyment* is responsible for the full lifecycle management of the pods, this includes in service update/upgrade.
+
+None of this is of much help however when implementing network service functions such as VNFs/CNFs that requires multi networking and network orchestartion
+
+<a name="3.5.7.2"></a>
+#### 3.5.7.2 Multi networking and Orchestration
+Kubernets does curently not in it self support multi networks, pod multi network attachments or network orchestration. This is supported by using a chain loading [*Container Network Interface*](https://github.com/containernetworking/cni) plugin such as [Multus:](https://github.com/k8snetworkplumbingwg/multus-cni).
+A considreable effort is being invested to add better network support to kubernetes, all such activities are coordinated through the kubernetes [*Network Special Interest Group*](https://github.com/kubernetes/community/tree/master/sig-network) and it's sub groups. One such group, the [*Network Plumbing Working Group*](https://github.com/k8snetworkplumbingwg/community) has produced the [Kubernetes Network Custom Resource Definition De-facto Standard](https://docs.google.com/document/d/1Ny03h6IDVy_e_vmElOqR7UdTPAG_RNydhVE1Kx54kFQ/edit). This document describes how secondary networks can be defined and attached to pods.
+
+This defacto standard defines among other things
+Definition | Description
+------------------|------------
+Kubernetes Cluster-Wide default network | A network to which all pods are attached following the current behaviour and requirements of Kubernetes, this done by attaching the *eth0* interface to the pod namespace.
+Network Attachment | A means of allowing a pod to directly communicate with a given logical or physical network. Typically (but not necessarily) each attachment takes the form of a kernel network interface placed in to the pod’s network namespace. Each attachment may result in zero or more IP addresses being assigned to the pod.
+NetworkAttachmentDefinition object | This defines resource object that describes how to attach a pod to a logical or physical network, the annotation name is *"k8s.v1.cni.cncf.io/networks"*
+Network Attachment Selection Annotation | Selects one or more networks that a pod should be attached to.
+
+Example: Define three network attachments and attach the three networks to a pod.
```

Comment:
I think this is too match detail.

Reply:
OK, lets discuss this, I rather put everything in and then cut instead of the other way arounf